pages/national_resources/gr_resources.md

```diff
@@ -1,65 +1,49 @@
-ref_to_main_resources: 
-  -  <!---REPLACE THIS with the tool id--->
 
 # List here tools and resources mainly relevant for the specific country
 national_resources: 
-  - name: <!---REPLACE THIS with the national resources about RDM in life sciences such as local instances of tools, guidelines or regulations--->
-    description:
-    how_to_access: <!--- REPLACE THIS free text to explain if credentials, login, specific affiliations, etc., are needed to access the resource or tool--->
-    instance_of: <!--- REPLACE THIS with the tool id of which this resource is an instance of, taken from the all tools and resources file --->
-    related_pages:
-      tool_assembly: [<!---REPLACE THIS with the page ID of the tool_assembly pages that you want to list here as related pages--->]
-      your_domain: [<!---REPLACE THIS with the page ID of the domain pages that you want to list here as related pages--->]
-      your_role: [<!---REPLACE THIS with the page ID of the your_role pages that you want to list here as related pages--->]
-      your_tasks: [<!---REPLACE THIS with the page ID of the your_tasks pages that you want to list here as related pages--->]
-    url:
-    registry:
-      biotools: <!--- DELETE ME if not needed --->
-      fairsharing: <!--- DELETE ME if not needed --->
-      tess: <!--- DELETE ME if not needed --->
----
-<!---All the resources added above will appear on the table at the bottom of the page--->
 
-<!---Following information for the page text--->
-<!---Use this template as guidance, all fields are optional. Feel free to modify any section if you think it is necessary--->
-<!---If the information is already in another resource, please include the link instead of duplicating information--->
-<!---Please focus on resources that are relevant for the whole country for life sciences--->
+  - name: Genomic Data Infrastructure (GDI)
+    description:  Infrastructure for secure genomic data management, including storage, discovery, access, and reception.
+    how_to_access:  Login via Life Science (LS) Login required to access ELIXIR-Greece GDI Portal.
+    url: https://login.gdi.elixir-greece.org/
+
+  - name: ARGOS
+    description:  ARGOS is an open, extensible service based on OpenCDMP software that simplifies the creation, management, and validation of research plans, including Data Management Plans (DMPs) and Software Management Plans (SMPs). It integrates with the OpenAIRE Graph, applying FAIR principles to ensure that research outputs are Findable, Accessible, Interoperable, and Reusable. ARGOS also supports research administration by streamlining workflows, enabling researchers, data stewards, and institutions to collaborate effectively in managing both research data and software. With curated templates and tools for planning, validation, and monitoring, ARGOS helps drive best practices in Open Science.
+    how_to_access:  OpenAIRE AAI
+    url: https://argos.openaire.eu/splash/
+    ref_to_main_resouces: argos
```

Comment:
```suggestion
```

This can go away since you can attribute your contributions like this: https://github.com/elixir-europe/rdmkit/pull/1568/files#r1963639186